pkgs/by-name/ae/aemu/package.nix

```diff
@@ -0,0 +1,28 @@
+{ clangStdenv
+, fetchFromGitiles
+, cmake
+, lib
+}:
+
+clangStdenv.mkDerivation {
```

Comment:
Why clang?

Reply:
The project uses clang-specific compiler flags.

Reply:
*Update* looks like it now also builds with gcc.